src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -238,7 +238,7 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
                       lgOffset={isColumnLayout ? null : 1}
                       xlOffset={isColumnLayout ? null : 1}
                     >
-                      {featuredArtists.length && (
+                      {featuredArtists.length > 0 && (
```

Comment:
I'm not sure if lodash's `take` guards against that but is there any chance featuredArtists could be `null`?

Reply:
`take` returns an empty array if passed `null`.

Reply:
But the more I think about it I think this is pointing to a bigger issue because in theory collections should have artworks and in the case of `/collection/andy-warhol-jean-michel-basquiat` there are works in staging but there aren't any on production.